tests/service/ProjectAnalysisTests.fs

```diff
@@ -4505,10 +4505,16 @@ module Project35b =
     open System.IO
 
     let fileName1 = Path.ChangeExtension(Path.GetTempFileName(), ".fsx")
+#if DOTNETCORE
+    let fileSource1 = """
+#r "System.Private.CoreLib.dll"
```

Comment:
So ... this is fine for a test but not as a long term solution.  We will have a decent solution for this eventually.

A preferable approach for tests would be to use:
https://github.com/Microsoft/visualfsharp/blob/master/tests/scripts/fsci.fsx
with a project.json file identifying dependencies.

Kevin